tapir/shifts/apps.py

```diff
@@ -61,6 +61,7 @@ def register_sidebar_links(cls):
             display_name="Solidarity Shifts",
             material_icon="favorite",
             url=reverse_lazy("shifts:solidarity_shifts"),
+            required_feature_flag="feature_flag_solidarity_shifts",
```

Comment:
Can you create a constant in `tapir/core/config.py` and use that where relevant?
(I'm suggesting other places in other comments)